tests/integration_tests.py

```diff
@@ -0,0 +1,49 @@
+import requests
```

Comment:
I moved the file into the `tests` directory. `pytest` won't run them automatically (which we don't want) because they are not named something like `test_`, so this should be fine. I like the approach of externalizing the tests into a separate module, and it's well coded.